configuration/reports/reportdescriptors/dataexports/mchProvider.yml

```diff
@@ -14,6 +14,10 @@ designs:
       "dateFormat": "dd-MMM-yyyy HH:mm:ss"
 config:
   category: "dataExport"
+  countries:
+    - "HAITI"
+    - "LIBERIA"
+    - "SIERRA_LEONE"
```

Comment:
@mogoodrich can you sanity check that this is what I need to do to remove this report from Mexico?

Reply:
I mean, that seems correct as I recall but don't know how much I should trust my memory... :)